### ISSUE TITLE
Spear Mastery Feat's upgraded damage die not applied in offhand

### ISSUE THREAD

**Issue by onlygamesandlife** [SolastaMods/SolastaUnfinishedBusiness#2743]
![Capture](https://user-images.githubusercontent.com/106723182/225545833-cd1274db-9d37-45ff-9cb5-6977ed5426b1.PNG)
Title.
Basically, I wanted to make a dual spear wielding character and I noticed that my offhand spear damage die was still d6, both in a campaign and during character creator/level/respec. This was happening for all the spears i checked (+1, +2, black viper, doom, doom tipped).

**Comment by Lordbat75** [comment]
The spear has a base damage of 1d6 and ONLY when equipped versatile way (no other weapon/shield/torch.., empty off-handed) the damage becomes 1d8. The weird thing is that in that image you do 1d8 dmg not empty off-handed with your main attack...
Provided you have the propper _**ambidextry**_ feat you can wear 2 spears (they are not light weapons), each of them will do 1d6 dmg

**Comment by onlygamesandlife** [comment]
the spear mastery feat added by unfinished business raises the one-handed spear die from 1d6 to 1d8, and versatile way from 1d8 to 1d10, which is why the main hand shows 1d8 despite having another spear equipped in the offhand. this happened with in all situations with or without two weapon fighting and ambidexterity. by taking the spear mastery feat, both spears should be upgraded to 1d8 because they are both being used one-handed (and it does properly upgrade from d8 to d10 from spear mastery during the versatile use way).

**Comment by Lordbat75** [comment]
That's right, you did not comment about the _spear mastery_ feat before, saying that, you are right, the damage should be 1d8 with that feat
